Source/VideoBlockViewController.swift

```diff
@@ -56,11 +61,20 @@ class VideoBlockViewController : UIViewController, CourseBlockViewController, St
                         if let video = block.type.asVideo, video.isYoutubeVideo,
                             let url = block.blockURL
                         {
-                            self?.showYoutubeMessage(url: url)
+                            if (self?.environment.config.youtubeConfig.enabled)!{
+                                let youtubeVideo = self?.environment.interface?.stateForVideo(withID: self?.blockID, courseID : self?.courseID)
```

Comment:
There's some apparent duplication here between 65+66 and 74+78 - should they be the same

Reply:
check